server/routes/profile.js

```diff
@@ -0,0 +1,23 @@
+import express from 'express';
+import Profiles from '../controllers/Profiles';
+import middlewares from '../middlewares';
+
+const {
+  verifyToken,
+  multerUploads,
+  checkUserVerification,
+  getSessionFromToken
+} = middlewares;
+
+const route = express.Router();
+
+route.patch(
```

Comment:
The validation middleware is not included here. Please add it to prevent getting a server when users try to enter invalid values when making their requests.